packages/runtime/container-runtime/api-report/container-runtime.api.md

```diff
@@ -218,6 +218,8 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents
     // (undocumented)
     get attachState(): AttachState;
     // (undocumented)
+    readonly baseLogger: ITelemetryBaseLogger;
```

Comment:
Please make sure all of the new members here that are `(undocumented)` get documentation. For anything inheriting from a base interface/class, feel free to use `{@inheritDoc Foo}`.

Reply:
Added document here